gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManager.java

```diff
@@ -105,7 +105,7 @@ public void updateTasks(Event event) {
                 LocalDateTime currentTime = toUtc(interval[0], processProperties.getTimezone());
```

Comment:
if you are always  expecting a UTC date from front-end use directly OffsetDateTime.parse("your-date-time").
If you agree, then i suggest to remove the timezone from configs

Reply:
Updated with new version of gridcapa-data-bridge